internal/serve/serve.go

```diff
@@ -155,6 +167,47 @@ func initHandlerDeps(cfg Configs) (handlerDeps, error) {
 	}
 	go ensureChannelAccounts(channelAccountService, int64(cfg.NumberOfChannelAccounts))
 
+	// TSS
+	txServiceOpts := tssservices.TransactionServiceOptions{
+		DistributionAccountSignatureClient: cfg.DistributionAccountSignatureClient,
+		ChannelAccountSignatureClient:      cfg.ChannelAccountSignatureClient,
+		HorizonClient:                      &horizonClient,
+		BaseFee:                            int64(cfg.BaseFee), // Reuse horizon base fee for RPC??
+	}
+	tssTxService, err := tssservices.NewTransactionService(txServiceOpts)
+	if err != nil {
+		return handlerDeps{}, fmt.Errorf("instantiating tss transaction service: %w", err)
+	}
+	httpClient := http.Client{Timeout: time.Duration(30 * time.Second)}
+	rpcService, err := services.NewRPCService(cfg.RPCURL, &httpClient)
+	if err != nil {
+		return handlerDeps{}, fmt.Errorf("instantiating rpc service: %w", err)
+	}
+
+	// re-use same context as above??
+	store := tssstore.NewStore(dbConnectionPool)
+	txManager := tssservices.NewTransactionManager(tssservices.TransactionManagerConfigs{
+		TxService:  tssTxService,
+		RPCService: rpcService,
+		Store:      store,
+	})
+	tssChannelConfigs := tsschannel.RPCCallerChannelConfigs{
+		TxManager:     txManager,
+		Store:         store,
+		MaxBufferSize: cfg.RPCCallerServiceChannelBufferSize,
+		MaxWorkers:    cfg.RPCCallerServiceChannelMaxWorkers,
+	}
+	rpcCallerServiceChannel := tsschannel.NewRPCCallerChannel(tssChannelConfigs)
```

Comment:
Shall we initialize all services the same way for better readability? Same for the `NewTransactionService` above.
```suggestion
	rpcCallerServiceChannel := tsschannel.NewRPCCallerChannel(tsschannel.RPCCallerChannelConfigs{
		TxManager:     txManager,
		Store:         store,
		MaxBufferSize: cfg.RPCCallerServiceChannelBufferSize,
		MaxWorkers:    cfg.RPCCallerServiceChannelMaxWorkers,
	})
```

Reply:
Yeap, these are all being initialized the same now